src/deploy.cmd.js

```diff
@@ -286,15 +379,59 @@ class DeployCommand extends AbstractCommand {
     };
     const openwhisk = ow(owoptions);
 
-    const scripts = [path.resolve(__dirname, 'openwhisk', 'static.js'), ...glob.sync(`${this._target}/*.js`)];
+    // get the list of scripts from the info files
+    const infos = [...glob.sync(`${this._target}/*.info.json`)];
```

Comment:
Do we have documentation for these `*.info.json` files?

Reply:
no. it is written in the RawPackager. I will add some documentation.

```
    // write info
    const bundleDir = path.dirname(this.bundle.name);
    const info = {
      main: path.basename(this.bundle.name),
      requires: Object.keys(deps)
        .map(dep => path.relative(bundleDir, path.resolve(bundleDir, dep))),
    };
    const infoName = path.resolve(bundleDir, `${path.basename(this.bundle.name, '.js')}.info.json`);
    await fs.writeJSON(infoName, info, { spaces: 2 });
```